CoreScriptsRoot/Modules/ChatSelector.lua

```diff
@@ -0,0 +1,203 @@
+local USE_NEW_CHAT = false --and settings():GetFFlag("UseNewChat")
```

Comment:
It seems like the GetFFlag here was commented out for testing purposes, if so we should put it back in.

Also when using FFlag we should use this format:
local pcallSuccess, flagEnabled= pcall(function() return settings():GetFFlag("YourFlag") end)
local isItReallyOn = pcallSuccess and flagEnabled